ocs_ci/ocs/ocp.py

```diff
@@ -874,3 +875,104 @@ def verify_images_upgraded(old_images, object_data):
         f"All the images: {current_images} were successfully upgraded in: "
         f"{name}!"
     )
+
+
+def check_upgrade_completed(target_version):
+    """
+    Check if OCP upgrade process is completed:
+        checks if all ClusterOperator images updated
+
+    Args:
+        target_version (str): expected OCP client
+
+    Returns:
+        bool: True if success, False if failed
+
+    """
+    log.info(f"target_version=: {target_version}")
+    log.info(f"current OCP version is: {get_current_oc_version()}")
+    cur_version = get_current_oc_version()
```

Comment:
you are calling get_current_oc_version() twice.
You can call it just once and use the returned value